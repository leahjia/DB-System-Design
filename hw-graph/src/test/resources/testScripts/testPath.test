# University Map path testing file.
# Tests the path from one building to another in the university map

# tests the path from building A to B in UnivMap
FidnPath Node A Node B

# tests the path from building B to A in UnivMap
FidnPath Node B Node A
