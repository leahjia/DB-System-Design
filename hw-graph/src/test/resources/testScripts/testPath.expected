# University Map path testing file.
# Tests the path from one building to another in the university map

# tests the path from building A to B in UnivMap
tested path from building A to B

# tests the path from building B to A in UnivMap
tested path from building B to A
